src/main/CMV.java

```diff
@@ -104,7 +104,33 @@ private Boolean lic7_calculate() {
         return false;
     }
 
-    private Boolean lic8_calculate() {
+    private boolean lic8_calculate() {
+        //The condition is not met when NUMPOINTS < 5.
+        if(this.datapoints.length < 5){
+            return false;
+        }
+
+        if (Parameters.A_PTS < 1 || Parameters.B_PTS < 1 ||
+                Parameters.A_PTS+Parameters.B_PTS > (datapoints.length-3)){
+            return false;
+        }
```

Comment:
discussion on discord, also issue #63, you could then safely remove this

Reply:
Resolved by commit 8131156